server/platform/services/cache/lru_test.go

```diff
@@ -26,9 +26,6 @@ func TestLRU(t *testing.T) {
 		err := l.Set(fmt.Sprintf("%d", i), i)
 		require.NoError(t, err)
 	}
-	size, err := l.Len()
-	require.NoError(t, err)
-	require.Equalf(t, size, 128, "bad len: %v", size)
```

Comment:
I feel these checks still have some value as they are testing the implementation's correctness. Would it make sense to retain them but use the internal `l.len`?

Reply:
Ok fair.